README.md

```diff
@@ -15,24 +15,56 @@ The schema is intended to support different use cases that EU Member States requ
 The schema in conjunction with business rules (which may also be specific to a Member State) shall ensure conformity to the EU legislation, whereby the burden of conformity lies with the business rules and the DCC JSON schema plays a supporting role in allowing the data to be serialized and deserialized in a flexible, yet structured, manner.
 
 
-## Files
+## Schema files
 
 The main schema file is:
 
-- DCC.schema.json
+- [DCC.schema.json](./DCC.schema.json)
 
 With the definitions in:
 
-- DCC.Core.Types.schema.json
-- DCC.Types.schema.json
-- DCC.ValueSets.schema.json
+- [DCC.Core.Types.schema.json](./DCC.Core.Types.schema.json)
+- [DCC.Types.schema.json](./DCC.Types.schema.json)
+- [DCC.ValueSets.schema.json](./DCC.ValueSets.schema.json)
 
 For easy testing, they are combined in:
 
-- DCC.combined-schema.json
+- [DCC.combined-schema.json](./DCC.combined-schema.json)
+
+The rule in the [Makefile](./Makefile) for this target performs the combining operation.
+The [`merge.sh` script](./merge.sh) also performs this operation, but without reformatting and validating the resulting JSON.
+This can be useful for development purposes, but make sure to also run `make` prior to committing.
+
+_Note:_ the first 4, non-combined schema files are always leading.
+
+
+### Value sets
+
+The [value set JSON files](./valuesets) should be validated against the [`valueset.json` schema file](./valueset.json).
+The value set file [`test-manf.json`](./valuesets/test-manf.json) can be generated by running the following CLI command:
+
+    $ source valuesets/source-update/update-rat.sh > valuesets/test-manf.json
+
+The `update-rat.sh` shell script retrieves the required information from https://covid-19-diagnostics.jrc.ec.europa.eu/.
+This is primarily meant as a tool for whoever is managing the governance of the central deployment.
+In particular, this script is _not_ (to be) run by the Makefile, nor the CI.
 
-But the above 4 files are always leading.
 
 ## Implementation Notes
 
 A list of [Frequently Asked Questions](https://github.com/ehn-dcc-development/ehn-dcc-schema/wiki/FAQ) for implementors can be found in the [repository wiki](https://github.com/ehn-dcc-development/ehn-dcc-schema/wiki).
+
+
+## Installation
+
+The Makefile assumes the following tools to be installed:
+
+* [Python (version 3)](https://www.python.org/downloads/)
+* [GNU Make](https://www.gnu.org/software/make/) to execute the [Makefile](./Makefile).
+* [Node.js](https://nodejs.org/en/download/) which comes with the [NPM package manager](https://www.npmjs.com/) which is used to install a couple of dependencies.
+* [jq](https://stedolan.github.io/jq/)
+
+The Makefile also relies on the [`ajv` JSON Schema validator](https://ajv.js.org/), which is installed by the `install-ajv` rule in the Makefile:
+
+    $ make install-ajv
+
```

Comment:
There's nothing here telling me how to install on my system, as requested https://github.com/ehn-dcc-development/ehn-dcc-schema/pull/100#discussion_r648674396